app/workers/scm/create_repository_job.rb

```diff
@@ -43,7 +43,8 @@ def initialize(repository)
 
   def perform
     # Create the repository locally.
-    mode = config[:mode] || default_mode
+    # chmod requires integer value
+    mode = (config[:mode] || default_mode).to_i
```

Comment:
Are you sure that this will work for string based configurations?

I checked your `default_mode` and sensibly you have written it as octal. But this won't  work for strings:

````ruby
2.1.4 :003 > 0700.to_i
 => 448 
2.1.4 :004 > '0700'.to_i
 => 700 
````